preprocessing/DataLoader.py

```diff
@@ -13,7 +13,7 @@ def __init__(self, file_path):
             # Abstract
             if "abstract" in content:
                 for entry in content['abstract']:
-                    self.abstract.append(entry['text'])
+                    self.abstract.append(str(entry['text']).strip())
```

Comment:
@redhed976 This is where I trimmed the leading and trailing whitespace in the abstract. Can you do this to each column loaded in onanother branch? Then if the cell contains an empty string, which is what's leftover if only whitespace existed after a trim, replace the whitespace with NaN unless it's an abstract (already handled in dropNulls).